lib/logdna.rb

```diff
@@ -20,50 +20,24 @@ def initialize(key, opts={})
       @level = opts[:level] || 'INFO'
       @env = opts[:env]
       @meta = opts[:meta]
-      @@client = nil unless defined? @@client
-      
+
       endpoint = opts[:endpoint] || Resources::ENDPOINT
       hostname = opts[:hostname] || Socket.gethostname
       ip =  opts.key?(:ip) ? "&ip=#{opts[:ip]}" : ''
       mac = opts.key?(:mac) ? "&mac=#{opts[:mac]}" : ''
       url = "#{endpoint}?hostname=#{hostname}#{mac}#{ip}"
+      uri = URI(url)
```

Comment:
Looks like you're no longer handling `URI::ValidURIRequired` - is that OK?

Reply:
I tested this functionality, it does not work. URI::ValidURIRequired 
And I don't see why to check this validation, if the url is not valid for any reason the exception will be thrown.